osu.Framework/Testing/TestBrowser.cs

```diff
@@ -504,12 +504,43 @@ private void finishLoad(TestScene newTest, Action onCompletion)
                 {
                     string repeatSuffix = i > 0 ? $" ({i + 1})" : string.Empty;
 
-                    if (m.GetCustomAttribute(typeof(TestAttribute), false) != null)
+                    var methodWrapper = new MethodWrapper(m.GetType(), m);
+
+                    if (methodWrapper.GetCustomAttributes<TestAttribute>(false).FirstOrDefault() != null)
```

Comment:
typical nit - I'd either use

```suggestion
                    if (methodWrapper.GetCustomAttributes<TestAttribute>(false).SingleOrDefault() != null)
```

(`AllowMultiple = false` on `TestAttribute` allows this), or use `Any()`.